Userland/Services/WindowServer/ScreenBackend.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "ScreenLayout.h"
+#include <AK/Error.h>
+#include <AK/Span.h>
+#include <sys/ioctl_numbers.h>
```

Comment:
sys/ioctl_numbers and screen forward decl not needed with the current factoring

Reply:
leftovers from previous version, absolutely correct

Reply:
Actually no, sys/ioctl_numbers defines FBRect and friends so we need that.